Traceback (most recent call last):
  File "/home/tevinw/ragviz/backend/ragviz.py", line 21, in <module>
    pile_search = PileSearch()
  File "/home/tevinw/ragviz/backend/search/pile/search.py", line 20, in __init__
    for filename in os.listdir(directory):
FileNotFoundError: [Errno 2] No such file or directory: 'None/backend/search/pile/ranges/'
